src/types/BaseField.ts

```diff
@@ -11,12 +11,18 @@ export enum BaseFieldDataType {
 	BOOLEAN = 'boolean',
 }
 
+export enum BaseFieldScope {
+	PROPOSAL = 'proposal',
+	ORGANIZATION = 'organization',
```

Comment:
Is the thought here that BaseFieldScope.ORGANIZATION is being used to relate proposal base fields to organizations(i.e data from proposals that can be used to define an organization) because it will be used equivalent to actual organization base-fields (ones that are derived from organization data, whenever we flesh out the protocol to create/register a new organization). If so, great! But then, are we going to weight the fields from proposals that describe organizations equally to the fields from organizations directly? That seems a bit weird to me but I could certainly see it working.

Sorry for the dump here, code works great and all the development here(tests/db migrations) look great!

Reply:
The logic for how we decide which value to use is for future versions of us to decide -- the change in this PR doesn't really dictate what that logic should be.  I imagine we might have use cases for situations where a proposal value is newer and different, and a PDC interface to allow organization delegates to update the directly-linked data via some kind of interface.